src/helpers/data/load-meeting-data.js

```diff
@@ -487,16 +487,16 @@ function flattenDays(data) {
       indexes_to_remove.push(index);
       meeting.day.forEach(day => {
         meetings_to_add.push({
+          ...meeting,
```

Comment:
If I'm understanding the intent of `flattenDays` correctly, I'm not sure this function was ever working since the `...meeting` was overriding the `day` and `slug` set 👀

Reply:
haha 🤦 

the intent of this function is to allow people to have a JSON feed that has `day: [0, 1, 2]` or a Google Sheet that has day = `Sunday, Monday, Tuesday` -- but it's very infrequently used so i'm not surprised it was a lurking bug. glad you found it!